### ISSUE TITLE
How I can use real-time public path?

### ISSUE THREAD

**Issue by bernex** [webpack-contrib/file-loader#112]
I have publicPath in `APP.url` variable and take it from page.

I have tried: `<img src={ require("file-loader?publicPath="+APP.url+"!./del.svg") }/>`
and `<img src={ APP.url + require("file-loader!./del.svg") }/>`

In css very simple - I can use `./` public path. With img.src it is not possible. It s relative to page url. And I can't use <base> in my situation.

How I can solve this task? Thank you!

**Comment by AndreKR** [comment]
You seem to be using some kind of template language, you should mention that.

Anyway, have you tried adding
```javascript
__webpack_public_path__ = 'my_path';
```
or
```javascript
__webpack_public_path__ = window.asset_root;
```
at the top of your entrypoint?

**Comment by evilebottnawi** [comment]
@bernex Is the problem still there? Why use use `APP.url`? Your can use `file-loader?name=[name].[ext]&publicPath=assets/foo/&outputPath=app/images/` to configure output directory instead using `APP.url`.

**Comment by michael-ciniawsky** [comment]
Fixed by #81
